app/assets/src/components/common/DetailsSidebar/SampleDetailsMode/metadata_section.scss

```diff
@@ -5,12 +5,12 @@
   .title {
     color: $black;
     font-weight: 600;
-    letter-spacing: 1.45px;
-    font-size: $font-size-body;
+    letter-spacing: 1.3px;
```

Comment:
Can you declare a new mixin in typography for this?

Reply:
👍